static/js/apps/eval_rig/app.tsx

```diff
@@ -151,12 +153,24 @@ export function App(props: AppPropType): JSX.Element {
             <p>Signed in as {user.email}</p>
           </div>
         )}
-        {user && allQuery["1"] && allCall["1"] && (
-          <AppContext.Provider
-            value={{ doc, sheetId: props.sheetId, userEmail: user.email }}
-          >
-            <QuerySection query={allQuery["1"]} calls={allCall["1"]} />
-          </AppContext.Provider>
+        {user && allQuery[selectedQuery] && allCall[selectedQuery] && (
+          <>
+            <EvalList
```

Comment:
put <EvalList> within <AppContext.Provider> as well so sheetId and user can be obtained from that.

Reply:
good idea, done